controllers/keystoneendpoint_controller.go

```diff
@@ -138,7 +137,7 @@ func (r *KeystoneEndpointReconciler) Reconcile(ctx context.Context, req ctrl.Req
 				condition.SeverityWarning,
 				keystonev1.KeystoneAPIReadyNotFoundMessage,
 			))
-			util.LogForObject(helper, "KeystoneAPI not found!", instance)
+			logFromCtx.Info("KeystoneAPI not found!", "instance", instance)
```

Comment:
We don't need to log the KeystoneEndpoint manually, that is automatically logged by logger.
```
l.Info("KeystoneAPI not found!")
```